app/javascript/components/SwapPanel.js

```diff
@@ -52,7 +55,7 @@ class SwapPanel extends React.Component {
       <div className="text-sm sm:text-lg md:text-xl" >
         <div className="px-6 py-3 flex justify-between items-center">
           <p className="font-semibold tabular-nums">
-            { this.props.num_vouchers_remaining_today }
+            { this.props.num_vouchers_remaining_today } vouchers left today
           </p>
           <button onClick={this.handleToggle} type="button" className="underline text-indigo-900">
             { this.state.isOpen ? "+ view per hotel" : "- hide per hotel" }
```

Comment:
I swapped these two labels: "+ view per hotel" and "- hide per hotel", so that the user clicks "- hide" when they wish to hide the panel, and they click "+ view" when they wish to view the panel.

Reply:
Nice, thanks!